src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap

```diff
@@ -376,22 +369,15 @@ exports[`StoryPromo Container should render correctly for canonical 1`] = `
 
 @media (min-width:20rem) and (max-width:37.4375rem) {
   .c5 {
-    font-size: 1rem;
-    line-height: 1.25rem;
-  }
-}
-
-@media (min-width:37.5rem) {
-  .c5 {
-    font-size: 1rem;
-    line-height: 1.25rem;
+    font-size: 1.125rem;
+    line-height: 1.375rem;
   }
 }
 
 @media (min-width:37.5rem) {
   .c5 {
-    font-size: 1.125rem;
-    line-height: 1.375rem;
+    font-size: 1.25rem;
```

Comment:
If you could run `npm run updateSnapshots` - these snapshots should not change any longer.